config/Shared/config_default.php

```diff
@@ -591,7 +591,7 @@
 $config[OmsConstants::PROCESS_LOCATION] = [
     OmsConfig::DEFAULT_PROCESS_LOCATION,
     APPLICATION_ROOT_DIR . '/vendor/spryker-eco/payone/config/Zed/Oms',
-    APPLICATION_ROOT_DIR . '/vendor/spryker/spryker/Bundles/Payment/config/Zed/Oms',
+    APPLICATION_ROOT_DIR . '/vendor/spryker/payment/config/Zed/Oms',
```

Comment:
Thats not good... This will always create merge conflicts in the future
Paths should be made more agnostic IMO in future releases!

Fow now I guess we have to continue with this.

Reply:
We got strong NO from @stereomon 
we suggested to use deprecated config `$config[KernelConstants::SPRYKER_ROOT]` (which is used in this file already)

Reply:
Well, using the deprecated path is worse
There are ways to make this agnostic on both sides with correct path (correct is always the one final in suite)
and only overwrite or modify on suite nonsplit side
This is the only valid way

But again, for now I think we can continue and this should be a follow up as per our rules and guidelines.